### PR TITLE
Separable resolution of the identity

### PR DESCRIPTION
**Pull request by fdelesma**
This MR includes the implementation of the separable resolution of the identity to the Localized-Basis component of the GreenX library. The test compares the Global-RI fitting coefficients with the one obtained in this implementation

### CODE REVIEW

File: GX-LocalizedBasis/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,139 @@
+# **************************************************************************************************
+#  Copyright (C) 2020-2023 GreenX library
```
Comment (dgolze) [inline GX-LocalizedBasis/CMakeLists.txt:2]: can you please change the year here, too?

File: GX-LocalizedBasis/src/localized_basis_types.f90 [hunk 2]
```diff
@@ -0,0 +1,50 @@
+! ***************************************************************************************************
+!  Copyright (C) 2020-2023 GreenX library
```
Comment (dgolze) [inline GX-LocalizedBasis/src/localized_basis_types.f90:2]: please change year to 2024

File: GX-LocalizedBasis/test/conftest.py [hunk 3]
```diff
@@ -0,0 +1,143 @@
+"""
+! **************************************************************************************************
+!  Copyright (C) 2020-2023 GreenX library
```
Comment (dgolze) [inline GX-LocalizedBasis/test/conftest.py:3]: year!

File: GX-LocalizedBasis/test/test_gx_localized_basis.f90 [hunk 4]
```diff
@@ -0,0 +1,71 @@
+program test_gx_localized_basis
```
Comment (dgolze) [inline GX-LocalizedBasis/test/test_gx_localized_basis.f90:1]: hm, does this also need a copy-right banner?

**Review by dgolze — CHANGES_REQUESTED** [review]
Just small comments

**Review by dgolze — APPROVED** [review]
looks good

**Review by moritzleucke — APPROVED** [review]

**Pull request merged**